internal/distro/fedora/distro.go

```diff
@@ -560,6 +559,28 @@ func (t *imageType) PackageSets(bp blueprint.Blueprint, repos []rpmmd.RepoConfig
 		mergedSets[name] = t.getPackages(name)
 	}
 
+	for name, packages := range manifest.GetPackageSets() {
+		// TODO: remove hack when pipeline names and package set
+		// names are the same.
+		switch name {
+		case "os":
+			name = "packages"
+		case "ostree-tree":
+			name = "packages"
+		case "container-tree":
+			name = "container"
+		case "anaconda-tree":
+			name = "installer"
```

Comment:
You should use defined constants for package set names, instead of string literals.

Reply:
Indeed! I probably need to rethink this whole hack, probably don't want to introduce this kind of stop gap...